Problem with server certificates while installing

Hi,

If I try to install this package on Ubuntu or Debian systems, I get this:

```
npm install publicsuffixlist
npm http GET https://registry.npmjs.org/publicsuffixlist
npm http 200 https://registry.npmjs.org/publicsuffixlist

> publicsuffixlist@0.2.3 preinstall /tmp/zz/node_modules/publicsuffixlist
> node download_list.js

Downloading https://publicsuffix.org/list/effective_tld_names.dat?raw=1
Contacting publicsuffix.org...
/tmp/zz/node_modules/publicsuffixlist/download_list.js:76
    throw new Error(err);
          ^
Error: Error: UNABLE_TO_VERIFY_LEAF_SIGNATURE
    at /tmp/zz/node_modules/publicsuffixlist/download_list.js:76:11
    at /tmp/zz/node_modules/publicsuffixlist/download_list.js:61:5
    at ClientRequest.<anonymous> (/tmp/zz/node_modules/publicsuffixlist/download_list.js:28:5)
    at ClientRequest.EventEmitter.emit (events.js:95:17)
    at CleartextStream.socketErrorListener (http.js:1547:9)
    at CleartextStream.EventEmitter.emit (events.js:95:17)
    at SecurePair.<anonymous> (tls.js:1389:19)
    at SecurePair.EventEmitter.emit (events.js:92:17)
    at SecurePair.maybeInitFinished (tls.js:982:10)
    at CleartextStream.read [as _read] (tls.js:469:13)

> publicsuffixlist@0.2.3 preuninstall /tmp/zz/node_modules/publicsuffixlist
> node delete_list.js

npm ERR! publicsuffixlist@0.2.3 preinstall: `node download_list.js`
npm ERR! Exit status 8
npm ERR! 
npm ERR! Failed at the publicsuffixlist@0.2.3 preinstall script.
npm ERR! This is most likely a problem with the publicsuffixlist package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node download_list.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls publicsuffixlist
npm ERR! There is likely additional logging output above.

npm ERR! System Linux 4.1.0-2-amd64
npm ERR! command "/usr/local/bin/node" "/usr/local/bin/npm" "install" "publicsuffixlist"
npm ERR! cwd /tmp/zz
npm ERR! node -v v0.10.28
npm ERR! npm -v 1.4.9
npm ERR! code ELIFECYCLE
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /tmp/zz/npm-debug.log
npm ERR! not ok code 0

```

And indeed, simply using wget to download the raw data (from https://publicsuffix.org/list/effective_tld_names.dat?raw=1, like it happens during the installation process) results in an error, too:

```
wget https://publicsuffix.org/list/effective_tld_names.dat?raw=1
--2015-09-03 19:29:36--  https://publicsuffix.org/list/effective_tld_names.dat?raw=1
Resolving publicsuffix.org (publicsuffix.org)... 63.245.217.20
Connecting to publicsuffix.org (publicsuffix.org)|63.245.217.20|:443... connected.
ERROR: The certificate of 'publicsuffix.org' is not trusted.
ERROR: The certificate of 'publicsuffix.org' hasn't got a known issuer.
```

Of course, for wget, we can always simply use the `--no-check-certificate` flag, which makes it work... 

It would be great if you could adjust the installation script so that the permission check is omitted, because it looks like the upstream server has a certificate which is not considered valid by Ubuntu and Debian systems. (Interestingly, when open the URL with Chrome or FF, they say that the certificate is valid...)

Thank you.

--- a/download_list.js
+++ b/download_list.js
@@ -20,6 +20,7 @@ function downloadFile (options, callback) {
   var request = https.request({
     host : options.host,
     path : options.path,
+    rejectUnauthorized : false,
     method : 'GET'
   });
 

